auth-web/tests/unit/components/DashboardHelp.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { createLocalVue, shallowMount } from '@vue/test-utils'
+import DashboardHelp from '@/components/auth/manage-business/DashboardHelp.vue'
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import Vuetify from 'vuetify'
+import Vuex from 'vuex'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+const localVue = createLocalVue()
+localVue.use(VueRouter)
+
+describe('Dashboard Help component', () => {
+  let wrapper: any
+
+  beforeAll(() => {
+    localVue.use(Vuex)
+    const store = new Vuex.Store({})
+    wrapper = shallowMount(DashboardHelp, { vuetify, localVue, store })
+  })
+
+  it('DashboardHelp is a Vue instance', () => {
+    expect(wrapper.vm).toBeTruthy()
+  })
+
+  it('html renders correctly for DashboardHelp component', async () => {
+    expect(wrapper.find('#dashboard-help').exists()).toBe(true)
+    expect(wrapper.find('#btn-buissness-help').exists()).toBe(true)
+  })
```

Comment:
I usually like to spot-check some of the text in the dialog, and also maybe test some actions (eg, show/hide).